plugins/uploader/download.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+set -e
+set -x
+
+if [ -z "$DEBUG" ]
+then
+    set +x
+fi
+usage()
+{
+cat << EOF
+usage: FAIRWINDS_TOKEN=secret $0 \
+    --organzation acme-co \
+    --cluster staging \
+    --file foo.json \
```

Comment:
I know this is just an example... but I think the file type is supposed to be `.yaml`?